src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -1089,26 +1089,6 @@ public Long execute(Jedis connection) {
   }
 
   @Override
-  public List<String> blpop(final String arg) {
```

Comment:
Oh, that's right. Thanks for finding!
Btw, we can do the trick, adding @Deprecated to methods and apply it to 2.7.